assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -199,33 +199,35 @@ export const DEFAULT_NOTIFICATIONS = {
 			VIEW_CONTEXT_ENTITY_DASHBOARD,
 		],
 		checkRequirements: async ( { select, resolveSelect, dispatch } ) => {
-			await Promise.all( [
-				// The getAdSenseLinked selector relies on the resolution
-				// of the getSettings() resolver.
-				resolveSelect( MODULES_ANALYTICS_4 ).getSettings(),
-				// The isModuleConnected() selector relies on the resolution
-				// of the getModules() resolver.
-				resolveSelect( CORE_MODULES ).getModules(),
-			] );
+			await resolveSelect( CORE_MODULES ).getModules();
 
 			const adSenseModuleConnected =
 				select( CORE_MODULES ).isModuleConnected( 'adsense' );
 
 			const analyticsModuleConnected =
 				select( CORE_MODULES ).isModuleConnected( 'analytics-4' );
```

Comment:
Let's use `resolveSelect` to get `isModuleConnected` instead of pre-resolving `getModules`. These can be awaited directly as they have the necessary resolvers.